mobile/components/DeviceConnectionStatusBar.tsx

```diff
@@ -8,12 +8,26 @@ const DeviceConnectionStatusBar = () => {
   const [deviceHeartBeat, setDeviceHeartBeat] =
     useRecoilState(DeviceHeartBeatState);
   const [connected, setConnected] = useRecoilState(DeviceConnectionState);
+  const [alertActive, setAlertActive] = useState(false);
 
   useEffect(() => {
     const interval = setInterval(() => {
       const newConnectionStatus = deviceHeartBeat + 15_000 > Date.now();
-      if (connected && !newConnectionStatus) {
-        Alert.alert("Your Sentinel bike tag has disconnected!");
+      if (connected && !newConnectionStatus && !alertActive) {
```

Comment:
don't actually need alertActive, just need to add `connected` in deps for this useEffect

Reply:
o okok thanks will do